### PR TITLE
Update message and inactivity intervals for stale issues bot

### PR DESCRIPTION
**Pull request by Mikejo5000**
Getting ready for shutdown of GH issues, currently targeting 6/30/24

### CODE REVIEW

**Comment by Mikejo5000** [comment]
@nkolev92 Nikolche, please take a look. I adjusted the message and inactivity periods to get the open issue count down significantly prior to end of June.

File: .github/workflows/stale-issues.yml [hunk 1]
```diff
@@ -22,13 +22,13 @@ jobs:
       - uses: actions/stale@v9
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-          stale-issue-message: 'Thanks for sharing your feedback. We apologize for our delayed response. Due to the sheer volume of feedback in our repo, we have decided to prioritize the newest issues and are automatically closing our oldest issues with an Issues bot. If you believe your feedback is still actionable, then either respond directly in this issue or open a new documentation issue and we’ll review. If no activity occurs in the next 14 days, this issue will be closed.'
+          stale-issue-message: 'Thanks for sharing your feedback. We apologize for our delayed response. Throughout 2024 we are phasing out Git issues as the feedback mechanism on content and replacing it with a new feedback system. If you believe your feedback needs to be reviewed prior to closing, then respond directly in this issue and we’ll take a look at it. If no activity occurs in the next 7 days, this issue will be closed.'
```
Comment (donnie-msft) [inline .github/workflows/stale-issues.yml:25]: How can we avoid confusion with our product still indeed using GitHub for issues

```suggestion
          stale-issue-message: 'Thanks for sharing your feedback. We apologize for our delayed response. Throughout 2024 we are phasing out GitHub issues as the feedback mechanism on documentation content and replacing it with a new feedback system. If you believe your feedback needs to be reviewed prior to closing, then respond directly in this issue and we’ll take a look at it. If no activity occurs in the next 7 days, this issue will be closed.'
```

Comment (Mikejo5000) [reply .github/workflows/stale-issues.yml:25]: Yes, it's good to differentiate.

**Comment by Mikejo5000** [comment]
@donnie-msft It looks like Nikolche is unavailable for some time. Can you approve the PR?

**Review by donnie-msft — APPROVED** [review]

**Comment by Mikejo5000** [comment]
@donnie-msft Thanks!

**Comment by donnie-msft** [comment]
@Mikejo5000 just want to clarify the PR description here, is there a migration plan for our issues into the new platform?
What I see today are many many issues that have been opened for years that are now going to be closed within 1 week. When they are closed, will you be reopening them in the new issue system? If the action of this PR is shutting down community feedback about docs, then we may need to come up with a plan.

**Comment by Mikejo5000** [comment]
@donnie-msft Nikolche was in the process of triaging and migrating issues he was concerned about. He was going to migrate them to the NuGet product repo. I'm not sure the status of that. We can be sure to find out before turning off the issues completely. Some writing teams are converting remaining open issues to DevOps User Stories, but I believe his preference was to move them to the other repo.

**Comment by donnie-msft** [comment]
> @donnie-msft Nikolche was in the process of triaging and migrating issues he was concerned about. He was going to migrate them to the NuGet product repo. I'm not sure the status of that. We can be sure to find out before turning off the issues completely. Some writing teams are converting remaining open issues to DevOps User Stories, but I believe his preference was to move them to the other repo.

Thanks for following up! I will review this with Nikolche once he's back and let you know before June 30.

**Comment by nkolev92** [comment]
I migrated the critical issues at the end of May. For some reason, the deadline was beginning of June, not end of June.

**Pull request merged**